Serially setup apps before deploying from directory (#2115)
livebook-dev/livebook: README.md

    Livebook instance within the cloud provider platform.

  * LIVEBOOK_APPS_PATH - the directory with app notebooks. When set, the apps
    are deployed on Livebook startup with the persisted settings. Password-protected
    notebooks will receive a random password, unless LIVEBOOK_APPS_PATH_PASSWORD

>     is set. When deploying on top of the Livebook Docker image, you can run apps
>     setup at build time by with `RUN /app/bin/run_apps_setup.sh`, this way all
>     dependencies are cached and starting the apps is faster.

Reviewer: ```suggestion
    is set. When deploying using Livebook's Docker image, you can warm-up
    your apps at build time with `RUN /app/bin/run_apps_setup.sh`, this way all
    dependencies are cached and starting your apps is faster.
```
